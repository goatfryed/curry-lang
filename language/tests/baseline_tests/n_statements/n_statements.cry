printf("Hello World!\n")
printf("Greetings from curry-lang")