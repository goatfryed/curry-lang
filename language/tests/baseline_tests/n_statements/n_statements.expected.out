"Hello World!\n""Greetings from curry-lang"